docs/internals/deprecation.txt

```diff
@@ -23,6 +23,9 @@ details on these changes.
 * The ``django.contrib.postgres.aggregates.mixins.OrderableAggMixin`` class
   will be removed.
 
+* The condition that ``per_page`` is less than or equal to ``orphans``
+  in ``django.core.paginator.BasePaginator`` is no longer allowed.
```

Comment:
We should mention Paginator and AsynPaginator rather than "BasePaginator" (as BasePaginator is an internal class)